components/gitpod-protocol/src/usage.ts

```diff
@@ -20,7 +20,7 @@ export interface BillableSession {
 
     workspaceType: BillableWorkspaceType;
 
-    // "standard" or "XL"
+    // "standard" or "large"
```

Comment:
We should drop this comment, as it's not accurate. For our use-cases, the class is just a string and we don't necessarily know much more about the meaning of it - for self-hosted it may even be entirely different workspaceClasses

Reply:
Ah, I see. Because the classes are determined by the workspace team?

Reply:
In part, but in general the idea behind classes is that for self-hosted, you should be able to customize (and name) them for the needs of each installation. Some may need Data Science workloads, some just tiny workspaces and we don't really know ahead of time what it will be

Reply:
> for self-hosted, you should be able to customize (and name) them

Good point!